app/views/partial/_user_profile.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="user_container">
+  <div class='user_photo'>
+    <% if user.photo.present? && user.photo.match(/.(jpg|jpeg|png|gif)$/i) %>
+      <%= image_tag(user.photo, alt: user.name) %>
+    <% else %>
+      <%= image_tag('default_user.png', alt: user.name) %>
+    <% end %>
+  </div>
+  <div class="user_card">
+    <a href="/user/<%= user.id %>" class="user_name"> <%= user.name %> </a>
+    <p class="posts_counter">Number of posts: <%= user.posts_counter %></p>
+  </div>
+</div>

```

Comment:
> * [ ]  Well done for using a partial 👏 However, the user's image is currently not being displayed on the UI instead of using a correct image URL. Kindly address this issue 🐛
> 
> Here's what I have in the console 👇	Here's the UI 👇
> ![image](https://user-images.githubusercontent.com/106119105/282240843-fe3e96be-5da2-4504-8b30-6254d0645ad5.png)	![image](https://user-images.githubusercontent.com/106119105/282240888-e7fc4dc9-e182-4090-8889-aec5fc262725.png)
> * [ ]  Also, the `posts_counter` is not displaying the number of posts we have 🐛 Currently, the user _Tom_ has 4 posts, however we are still getting **_Number of posts: 0_**
> 
> ![image](https://user-images.githubusercontent.com/106119105/282241008-edb39bc7-9c1e-4b25-90b1-bbc488dcd800.png)


I intentionally used a dummy sample image to represent users for now because it is just a wireframe. Plus the project requirements is not concerned with css.

Reply:
Hello @Mike47ip, I appreciate that 😃 However, I've confirmed it with the Senior Reviewer, and it's required. If you have any confusions regarding this aspect, please feel free to reach out to me on slack. You can also request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

_Keep rocking 🚀_